### ISSUE TITLE
All Multiplayer suggestions as whole Series

### ISSUE THREAD

**Issue by eweninja** [finepointcgi/FinepointCGI-Suggestions#25]
As said in other suggestions, all multiplayer ideas can be packed into one series. The easiest way to go would be creating one, simple project and then convert it to different multiplayer suggestions or to have one starting project and starting from it.

1. [LAN Multiplayer](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/15)
2. [Client-Server with Godot Networking](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/4)
3. [EOS Multiplayer](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/23)
4. [Playfab Multiplayer](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/24)
5. [Firebase (Realtime)](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/26)
6. [WebRTC with Websocket Signaling Server](https://github.com/finepointcgi/FinepointCGI-Suggestions/issues/41)

and so on. 😊

**Comment by finepointcgi** [comment]
I agree... Im thinking of doing a pong project or something like that thats simple and easy to code then add multiplayer to it

**Comment by eweninja** [comment]
I've added the WebRTC with Signaling Server suggestion to the list above.
